wwwapp/settings.py

```diff
@@ -217,6 +218,7 @@
 ]
 
 CURRENT_YEAR = 2019
+WORKSHOPS_START_DATE = datetime.date(CURRENT_YEAR, 7, 1)
```

Comment:
This should be refactored at some point along with CURRENT_YEAR

Reply:
Not in the scope of this issue